app/conduct/conduct.js

```diff
@@ -0,0 +1,8 @@
+angular.module('gdgXBoomerang')
+.controller('ConductController', function ($http, $sce, Config, NavService) {
+    var vm = this;
+    vm.loading = true;
+    NavService.setNavTab(6);
+    vm.chapter = Config.name;
+    vm.lead = Config.lead;  
```

Comment:
Wouldn't it be better if we provide a link to the organisers tab in HTML file instead of finding the lead in controller? In that way we don't have to show the same data in different places, I guess